receiver/sqlqueryreceiver/config.go

```diff
@@ -103,6 +106,16 @@ func (c MetricCfg) Validate() error {
 	if c.DataType == MetricTypeGauge && c.Aggregation != "" {
 		errs = multierr.Append(errs, fmt.Errorf("aggregation=%s but data_type=%s does not support aggregation", c.Aggregation, c.DataType))
 	}
+	if c.StartTsColumn != "" {
+		if _, err := strconv.ParseUint(c.StartTsColumn, 10, 64); err != nil {
```

Comment:
Why are you trying to parse column names into integers? This doesn't look right. My understanding is that the `start_ts_column` and `stop_ts_column` can contain a string, which is the name of a column. Trying to parse this into a string doesn't make much sense.

Reply:
Sincere apologies for the confusion. You're right this is not required. Omitting it.